packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```
@@ -0,0 +1,68 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Created by pyp2rpm-3.3.7
+%global pypi_name pulp-cli-deb
+
+Name:           %{?scl_prefix}python-%{pypi_name}
+Version:        0.0.1 
```

```suggestion
Version:        0.0.1
```
Nitpick: Trailing whitespace.